spec/factories.rb

```diff
@@ -97,4 +97,12 @@
   factory :specialization do
```

Comment:
It's more common to divide factories by different files. It's will be simpler for you when the quantity of factories increase.